src/main/java/nl/tudelft/semgroup4/StartScreenStateController.java

```diff
@@ -0,0 +1,203 @@
+
+import org.newdawn.slick.Color;
+import org.newdawn.slick.GameContainer;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.Image;
+import org.newdawn.slick.Input;
+import org.newdawn.slick.SlickException;
+import org.newdawn.slick.TrueTypeFont;
+import org.newdawn.slick.geom.Rectangle;
+import org.newdawn.slick.geom.Shape;
+import org.newdawn.slick.geom.Transform;
+import org.newdawn.slick.gui.MouseOverArea;
+import org.newdawn.slick.state.StateBasedGame;
+
+/**
+ * Controller that performs logic for the StartScreenState.
+ * 
+ * @author Pathemeous
+ *
+ */
+public class StartScreenStateController {
+
+    private ResourcesWrapper resources;
+
+    private static final String HIGHSCORE_TEXT = "HIGHSCORES";
+    private final TrueTypeFont typeFont;
+
+    /**
+     * Creates a new controller for a specific {@link StartScreenState}.
+     * 
+     * @param resources
+     *            {@link ResourcesWrapper} - The resources that this controller may use.
+     */
+    public StartScreenStateController(ResourcesWrapper resources) {
+        this.resources = resources;
+
+        Font font = new Font("Verdana", Font.BOLD, 36);
+        typeFont = resources.createFont(font, true);
+    }
+
+    /**
+     * Creates the HighScore button with the appropriate text.
+     * 
+     * @throws SlickException
+     *             If creating the {@link Image} goes wrong.
+     */
+    protected MouseOverArea createHighscoresButton(GameContainer container)
+            throws SlickException {
+        Shape shape =
+                new Rectangle(650, 650, typeFont.getWidth(HIGHSCORE_TEXT),
+                        typeFont.getHeight());
+        shape = shape.transform(Transform.createRotateTransform(6, 650, 650));
+        Image highScoreButton =
+                new Image(typeFont.getWidth(HIGHSCORE_TEXT), typeFont.getHeight());
+
+        shape =
+                new Rectangle(650, 650, typeFont.getWidth(HIGHSCORE_TEXT),
+                        typeFont.getHeight());
+        shape = shape.transform(Transform.createRotateTransform(6, 650, 650));
+
+        return new MouseOverArea(container, highScoreButton, shape);
+    }
+
+    /**
+     * Draws the text of the HighsScoresButton on the screen.
+     * 
+     * @param graphics
+     *            {@link Graphics} - The graphics that this method may use to draw.
+     */
+    protected void drawHighScoresButton(Graphics graphics) {
+        graphics.rotate(650, 650, 340);
+        typeFont.drawString(650, 650, HIGHSCORE_TEXT, Color.red);
+    }
+
+    /**
+     * Creates a new {@link SingleplayerGame}.
+     * 
+     * @param container
+     *            {@link GameContainer} - The state's Gamecontainer.
+     * @param mainApp
+     *            - {@link StateBasedGame} - The StateBasedGame to pass to the Game object.
+     * @return {@link Game} - A game object.
+     */
+    protected Game createSingleplayerGame(GameContainer container, StateBasedGame mainApp) {
```

Comment:
Could creating a player and adding its observer be done in a method which can be called in both createSinglePlayerGame and createMultiplayerGame?